spec.bs

```diff
@@ -784,3 +788,25 @@ specification adds another value for fenced frames:
 
 : "`fencedframe`"
 :: This [=navigable=] is displaying a <{fencedframe}>'s content
+
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
+
+The <a href=https://www.w3.org/TR/cssom-view-1>CSSOM View Module</a> defines scroll bubbling behavior
+through the [=scroll a target into view=] algorithm. Our specification modifies it to not allow
+scrolling a target into view if it is inside of a fenced frame.
+
+Modify the [=scroll a target into view=] algorithm to add 2 steps after step 1 that reads:
+
+2. If the <var ignore=''>target</var>-associated {{Document}}'s [=node navigable=]'s
+   [=navigable/traversable navigable=] is a [=fenced navigable container/fenced navigable=], terminate these steps.
+3. If the <var ignore=''>target</var>-associated {{Document}}'s [=Document/ancestor navigables=] contains
+   a [=navigable=] whose [=navigable/traversable navigable=] is a [=fenced navigable container/fenced navigable=],
+   terminate these steps.
+
+Modify the [=run the scroll steps=] algorithm to add a check after step 1.1 that reads:
+
+2. If <var ignore=''>target</var> is a {{Document}} whose [=node navigable=]'s
```

Comment:
This matches the impl behavior that doesn't propagate scroll events into a fenced frame. Is this the expected behavior? I wrote this just to match what we currently have in impl, but if this isn't what the behavior is supposed to be, we should file a crbug to fix.